package/preview/PKGBUILD

```diff
@@ -17,18 +17,21 @@ depends__apt=(
   "python3"
   "python3-setuptools"
   "service-discover"
+  "libcairo2-dev"
 )
 depends__rocky_8=(
   "pending-setups"
   "python38"
   "python38-setuptools"
   "service-discover"
+  "cairo-devel"
```

Comment:
I would try to remove it after the back merge with the develop branch. Test the package on RHEL-X